src/storyblok-auth-api/handle-requests/openidClient.ts

```diff
@@ -12,19 +12,19 @@ export type CreateOpenIdClient = (
 ) => BaseClient
 
 export const openidClient: CreateOpenIdClient = (params, region) => {
+  const defaultEndpoint = region
+    ? `${getRegionBaseUrl(region)}`
+    : 'https://app.storyblok.com'
+
+  const oauthEndpoint =
+    process.env['APP_CUSTOM_OAUTH_ENDPOINT'] ?? defaultEndpoint
```

Comment:
This library is built on functional programming principles: as such, we cannot read environmental variables. (@Dawntraoz recently removed some environmental variables that had accidentally been introduced earlier.) Instead, we should pass these variables as parameters. In this code, it should be passed via `params`.

If the client app wants to define this environmental variable, it can do so and pass it as a parameter to the app-extension-auth library